Tasks/JenkinsDownloadArtifacts/ArtifactDetails/ArtifactDetailsDownloader.ts

```diff
@@ -105,11 +103,36 @@ export class ArtifactDetailsDownloader {
         return defer.promise;
     }
 
-    private GetBuildIdIndex(startBuildId: number, endBuildId: number): Q.Promise<any> {
+    private GetBuildIdFromVersion(version: string, isMultibranchPipeline: boolean): number {
+        let buildId: number = NaN;
+
+        if (!!version) {
+            if (!isMultibranchPipeline) {
+                buildId = parseInt(version);
```

Comment:
assuming parseInt wouldn't fail if version is not an int?

Reply:
It doesn't it returns a NaN if it can't parses it.